HybridCar.hpp

```diff
@@ -1,19 +1,17 @@
+#pragma once
+#include "ElectricCar.hpp"
 #include "ElectricEngine.hpp"
+#include "PetrolCar.hpp"
 #include "PetrolEngine.hpp"
+#include <memory>
 
-class HybridCar
-{
-public:
-    HybridCar(PetrolEngine* petrolEng, ElectricEngine* electricEng);
-    ~HybridCar();
-    void turnLeft();
-    void turnRight();
-    void brake();
-    void accelerate(int speed);
-    void charge();
-    void refuel();
+class HybridCar : public PetrolCar, public ElectricCar {
 
-    PetrolEngine* petrolEngine_;
-    ElectricEngine* electricEngine_;
+public:
+  HybridCar(std::unique_ptr<PetrolEngine> petrolEng,
+            std::unique_ptr<ElectricEngine> electricEng);
+  ~HybridCar();
+  PetrolEngine *getPetrolEngine() const;
```

Comment:
Why getPetrolEngine only? Why not getElectricEngine?
BTW I don't see an implementation of this function.